### DISCUSSION TITLE
I wonder if we can use GPTQ quantized models for this? 7B HF models are too big even for 12Gb cards...

### DISCUSSION THREAD

**Discussion by barleyj21** [PromtEngineer/localGPT#58] [General]
7B HF models take more than 12Gb of memory, so is there a way to use GPTQ ones?

**Comment by PromtEngineer** [comment]
Its a good idea. I am testing them and also looking at adding support for different embeddings.

**Comment by PromtEngineer** [comment]
Just saw the new [accelerate ](https://github.com/huggingface/accelerate/releases/tag/v0.20.0) update. I think it will now be possible.

**Comment by LitenBuzzTh** [comment]
Support for GPTQ models would be awesome

**Comment by KonradHoeffner** [comment]
You can use GPTQ now.

**Reply by LitenBuzzTh** [reply]
yeah been using it for a while
